exporters/trace/README.md

```diff
@@ -82,7 +82,47 @@ before passing it into the TraceExporter constructor
 
 #### Java Versions
 Java 8 or above is required for using this exporter.
-  
+
+
+#### Special Attributes
+
+The Trace Exporter will add the following additional attributes onto all exported spans:
+
+- All `Resource` attributes that do not have a key name conflict with underlying `Span` attributes
+- `g.co/r/{resourcee_type}/{label}` attributes that correlate with GCP monitored resource.
+- A `g.co/agent` attribute that denotes the exporter used to write the Span.
+- `otel.scope.name` and `otel.scope.version`, as specified [by OpenTelemetry exporter requirements](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/sdk_exporters/non-otlp.md#instrumentationscope)
+
+#### Span Attribute relabelling
+
+The Trace Exporter attempts to convert from OpenTelemetry semantic conventions into Cloud Trace conventions.
+
+By default, the following span or event attributes will be converted as follows:
+
+| OpenTelemetry Attribute        | Cloud Trace Attribute   |
+|--------------------------------|-------------------------|
+| `http.host`                    | `/http/host`            |
```

Comment:
should golang stop doing conversion for service.name to the app-engine label by default?